checkbox-provider/lib/qat_tools-1.0.0/src/qat_tools/qatlib.py

```diff
@@ -298,7 +298,10 @@ def __init__(self, filter_devs = []):
       for pci_desc in pci_ids:
         pci_id = pci_desc.split(' ')[0]
         if (filter_devs is None) or (pci_id in filter_devs):
-          _devs.append(Qat4xxxDevice(device_id, pci_id))
+          try:
+            _devs.append(Qat4xxxDevice(device_id, pci_id))
+          except:
+            pass
```

Comment:
I think this will just catch any type of exception - I would suggest catching the type of exception you're expecting here for cases where it's okay to ignore, and then emit a warning to provide details to the user about why this is happening.

Reply:
fix proposed at : 681f25540be0add29382b55fc089959d34a8abfe